internal/exporter/httpsinkexporter/factory.go

```diff
@@ -31,10 +31,12 @@ const (
 
 // NewFactory creates a factory for httpsink exporter.
 func NewFactory() exporter.Factory {
+	sinkFactory := newSinkFactory()
```

Comment:
Does this ensure there's only one sink regardless of desired endpoint? What if multiples are wanted?

```
httpsink/one:
  endpoint: localhost:12345
httpsink/two:
  endpoint: localhost:23456
```

Reply:
This ensures there is one sink (http server) per exporter instance. Your example will start two sink servers. This was introduced so both metrics and spans start only one server per exporter instance. In the previous implementation they'd have tried to start the server twice for a single exporter instance.

Reply:
You're right. There is a bug in this logic. Will fix shortly.